blockstack/blockstack-android
# Add betaMode flag in BlockstackSession

This PR
* adds a flag `betaMode` in the constructor of `BlockstackSession`
* adds a test for beta mode

* fixes #132 

To test:
1. checkout branch `feature/beta-mode`
1. run instrumentation test `BlockstackSessionLoginTest` 
1. see that the beta browser was opened
